### PR TITLE
Region object add filed 'site' and get price based on the site

### PR DESCRIPTION
**Pull request by baixinsui**
Fixed https://github.com/eclipse-xpanse/xpanse/issues/1890

Based on https://github.com/eclipse-xpanse/xpanse/pull/1895

### CODE REVIEW

File: src/components/content/order/common/RegionSelection.tsx [hunk 1]
```diff
@@ -39,15 +40,22 @@ export const RegionSelection = ({
                 <Space wrap>
                     <Select
                         className={serviceOrderStyles.selectBoxClass}
-                        defaultValue={selectRegion}
-                        value={selectRegion}
+                        defaultValue={selectRegion.name + '(' + selectRegion.site + ')'}
```
Comment (swaroopar) [inline src/components/content/order/common/RegionSelection.tsx:43]: space missing between name and brackets.

File: src/components/content/order/common/RegionSelection.tsx [hunk 2]
```diff
@@ -39,15 +40,22 @@ export const RegionSelection = ({
...
+                        value={selectRegion.name + '(' + selectRegion.site + ')'}
                         onChange={(newRegion) => {
                             if (onChangeRegion) {
-                                onChangeRegion(newRegion);
+                                const regionName: string = newRegion.split('(')[0];
+                                const siteName: string = newRegion.split('(')[1].split(')')[0];
+                                const newRegionObject: Region = {
+                                    name: regionName.trim(),
+                                    site: siteName.trim(),
+                                    area: '',
```
Comment (swaroopar) [inline src/components/content/order/common/RegionSelection.tsx:52]: its better if we can send the area here instead of empty.

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: src/components/content/order/common/RegionSelection.tsx (hunk 2 above)
Comment (baixinsui) [reply src/components/content/order/common/RegionSelection.tsx]: Updated the area value with 'selectArea'

File: src/components/content/order/common/RegionSelection.tsx (hunk 1 above)
Comment (baixinsui) [reply src/components/content/order/common/RegionSelection.tsx]: Updated

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**